### ISSUE TITLE
2025-02-17, Papers for large language model, llm

### ISSUE THREAD

**Issue by 9tailwolf** [Arxiv-Sender/Arxiv-Sender#86]
# Today's Arxiv Papers

(1) [MM-RLHF: The Next Step Forward in Multimodal LLM Alignment](http://arxiv.org/abs/2502.10391v1), 2025-02-14

(2) [Aspect-Oriented Summarization for Psychiatric Short-Term Readmission Prediction](http://arxiv.org/abs/2502.10388v1), 2025-02-14

(3) [Enhancing Multilingual LLM Pretraining with Model-Based Data Selection](http://arxiv.org/abs/2502.10361v1), 2025-02-14

(4) [Agentic Verification for Ambiguous Query Disambiguation](http://arxiv.org/abs/2502.10352v1), 2025-02-14

(5) [Organize the Web: Constructing Domains Enhances Pre-Training Data Curation](http://arxiv.org/abs/2502.10341v1), 2025-02-14

(6) [Evaluating the Meta- and Object-Level Reasoning of Large Language Models for Question Answering](http://arxiv.org/abs/2502.10338v1), 2025-02-14

(7) [Process Reward Models for LLM Agents: Practical Framework and Directions](http://arxiv.org/abs/2502.10325v1), 2025-02-14

(8) [LLM-Powered Preference Elicitation in Combinatorial Assignment](http://arxiv.org/abs/2502.10308v1), 2025-02-14

(9) [Open-Source AI-Powered Optimization in Scalene: Advancing Python Performance Profiling with DeepSeek-R1 and LLaMA 3.2](http://arxiv.org/abs/2502.10299v1), 2025-02-14

(10) [Are Large Language Models the future crowd workers of Linguistics?](http://arxiv.org/abs/2502.10266v1), 2025-02-14

(11) [Large Language Models and Synthetic Data for Monitoring Dataset Mentions in Research Papers](http://arxiv.org/abs/2502.10263v1), 2025-02-14

(12) [VisCon-100K: Leveraging Contextual Web Data for Fine-tuning Vision Language Models](http://arxiv.org/abs/2502.10250v1), 2025-02-14

(13) [Efficient Zero-Order Federated Finetuning of Language Models for Resource-Constrained Devices](http://arxiv.org/abs/2502.10239v1), 2025-02-14

(14) [Do Large Language Models Reason Causally Like Us? Even Better?](http://arxiv.org/abs/2502.10215v1), 2025-02-14

(15) [Can Post-Training Quantization Benefit from an Additional QLoRA Integration?](http://arxiv.org/abs/2502.10202v1), 2025-02-14

(16) [Prediction hubs are context-informed frequent tokens in LLMs](http://arxiv.org/abs/2502.10201v1), 2025-02-14

(17) [MathConstruct: Challenging LLM Reasoning with Constructive Proofs](http://arxiv.org/abs/2502.10197v1), 2025-02-14

(18) [Translating Common Security Assertions Across Processor Designs: A RISC-V Case Study](http://arxiv.org/abs/2502.10194v1), 2025-02-14

(19) [SessionRec: Next Session Prediction Paradigm For Generative Sequential Recommendation](http://arxiv.org/abs/2502.10157v1), 2025-02-14

(20) [Semantica: Decentralized Search using a LLM-Guided Semantic Tree Overlay](http://arxiv.org/abs/2502.10151v1), 2025-02-14

(21) [Cooperative Multi-Agent Planning with Adaptive Skill Synthesis](http://arxiv.org/abs/2502.10148v1), 2025-02-14

(22) [Small Models, Big Impact: Efficient Corpus and Graph-Based Adaptation of Small Multilingual Language Models for Low-Resource Languages](http://arxiv.org/abs/2502.10140v1), 2025-02-14

(23) [ScamFerret: Detecting Scam Websites Autonomously with Large Language Models](http://arxiv.org/abs/2502.10110v1), 2025-02-14

(24) [NeuroXVocal: Detection and Explanation of Alzheimer's Disease through Non-invasive Analysis of Picture-prompted Speech](http://arxiv.org/abs/2502.10108v1), 2025-02-14

(25) [A novel approach to data generation in generative model](http://arxiv.org/abs/2502.10092v1), 2025-02-14

(26) [Enhancing Patient Acceptance of Robotic Ultrasound through Conversational Virtual Agent and Immersive Visualizations](http://arxiv.org/abs/2502.10088v1), 2025-02-14

(27) [DiSciPLE: Learning Interpretable Programs for Scientific Visual Discovery](http://arxiv.org/abs/2502.10060v1), 2025-02-14

(28) [ORI: O Routing Intelligence](http://arxiv.org/abs/2502.10051v1), 2025-02-14

(29) [A Survey on LLM-powered Agents for Recommender Systems](http://arxiv.org/abs/2502.10050v1), 2025-02-14

(30) [ViRAC: A Vision-Reasoning Agent Head Movement Control Framework in Arbitrary Virtual Environments](http://arxiv.org/abs/2502.10046v1), 2025-02-14

(31) [POI-Enhancer: An LLM-based Semantic Enhancement Framework for POI Representation Learning](http://arxiv.org/abs/2502.10038v1), 2025-02-14

(32) [Probabilistic Lexical Manifold Construction in Large Language Models via Hierarchical Vector Field Interpolation](http://arxiv.org/abs/2502.10013v1), 2025-02-14

(33) [ChatGPT and Deepseek: Can They Predict the Stock Market and Macroeconomy?](http://arxiv.org/abs/2502.10008v1), 2025-02-14

(34) [EmbBERT-Q: Breaking Memory Barriers in Embedded NLP](http://arxiv.org/abs/2502.10001v1), 2025-02-14

(35) [Decision Information Meets Large Language Models: The Future of Explainable Operations Research](http://arxiv.org/abs/2502.09994v1), 2025-02-14

(36) [Large Language Diffusion Models](http://arxiv.org/abs/2502.09992v1), 2025-02-14

(37) [X-Boundary: Establishing Exact Safety Boundary to Shield LLMs from Multi-Turn Jailbreaks without Compromising Usability](http://arxiv.org/abs/2502.09990v1), 2025-02-14

(38) [V2V-LLM: Vehicle-to-Vehicle Cooperative Autonomous Driving with Multi-Modal Large Language Models](http://arxiv.org/abs/2502.09980v1), 2025-02-14

(39) [LaRA: Benchmarking Retrieval-Augmented Generation and Long-Context LLMs - No Silver Bullet for LC or RAG Routing](http://arxiv.org/abs/2502.09977v1), 2025-02-14

(40) [Has My System Prompt Been Used? Large Language Model Prompt Membership Inference](http://arxiv.org/abs/2502.09974v1), 2025-02-14

(41) [Diverse Inference and Verification for Advanced Reasoning](http://arxiv.org/abs/2502.09955v1), 2025-02-14

(42) [A Preliminary Exploration with GPT-4o Voice Mode](http://arxiv.org/abs/2502.09940v1), 2025-02-14

(43) [Precise Parameter Localization for Textual Generation in Diffusion Models](http://arxiv.org/abs/2502.09935v1), 2025-02-14

(44) [MIR-Bench: Benchmarking LLM's Long-Context Intelligence via Many-Shot In-Context Inductive Reasoning](http://arxiv.org/abs/2502.09933v1), 2025-02-14

(45) [Granite Vision: a lightweight, open-source multimodal model for enterprise Intelligence](http://arxiv.org/abs/2502.09927v1), 2025-02-14

(46) [λScale: Enabling Fast Scaling for Serverless Large Language Model Inference](http://arxiv.org/abs/2502.09922v1), 2025-02-14

(47) [INF^2: High-Throughput Generative Inference of Large Language Models using Near-Storage Processing](http://arxiv.org/abs/2502.09921v1), 2025-02-14

(48) [AutoS$^2$earch: Unlocking the Reasoning Potential of Large Models for Web-based Source Search](http://arxiv.org/abs/2502.09913v1), 2025-02-14

(49) [The Ann Arbor Architecture for Agent-Oriented Programming](http://arxiv.org/abs/2502.09903v1), 2025-02-14

(50) [ChatIoT: Large Language Model-based Security Assistant for Internet of Things with Retrieval-Augmented Generation](http://arxiv.org/abs/2502.09896v1), 2025-02-14

(51) [ArchRAG: Attributed Community-based Hierarchical Retrieval-Augmented Generation](http://arxiv.org/abs/2502.09891v1), 2025-02-14

(52) [Video2Policy: Scaling up Manipulation Tasks in Simulation through Internet Videos](http://arxiv.org/abs/2502.09886v1), 2025-02-14

(53) [A Taxonomy of Linguistic Expressions That Contribute To Anthropomorphism of Language Technologies](http://dx.doi.org/10.1145/3706598.3714038), 2025-02-14

(54) [Solvable Dynamics of Self-Supervised Word Embeddings and the Emergence of Analogical Reasoning](http://arxiv.org/abs/2502.09863v1), 2025-02-14

(55) [Automated Hypothesis Validation with Agentic Sequential Falsifications](http://arxiv.org/abs/2502.09858v1), 2025-02-14

(56) [Port-LLM: A Port Prediction Method for Fluid Antenna based on Large Language Models](http://arxiv.org/abs/2502.09857v1), 2025-02-14

(57) [Efficient Multitask Learning in Small Language Models Through Upside-Down Reinforcement Learning](http://arxiv.org/abs/2502.09854v1), 2025-02-14

(58) [HealthGPT: A Medical Large Vision-Language Model for Unifying Comprehension and Generation via Heterogeneous Knowledge Adaptation](http://arxiv.org/abs/2502.09838v1), 2025-02-14

(59) [A Solver-Aided Hierarchical Language for LLM-Driven CAD Design](http://arxiv.org/abs/2502.09819v1), 2025-02-13

(60) [Statistical Coherence Alignment for Large Language Model Representation Learning Through Tensor Field Convergence](http://arxiv.org/abs/2502.09815v1), 2025-02-13

(61) [INJONGO: A Multicultural Intent Detection and Slot-filling Dataset for 16 African Languages](http://arxiv.org/abs/2502.09814v1), 2025-02-13

(62) [AgentGuard: Repurposing Agentic Orchestrator for Safety Evaluation of Tool Orchestration](http://arxiv.org/abs/2502.09809v1), 2025-02-13

(63) [Unit Testing Past vs. Present: Examining LLMs' Impact on Defect Detection and Efficiency](http://arxiv.org/abs/2502.09801v1), 2025-02-13

(64) [Co-designing Large Language Model Tools for Project-Based Learning with K12 Educators](http://dx.doi.org/10.1145/3706598.3713971), 2025-02-13

(65) [A Survey on LLM-based News Recommender Systems](http://arxiv.org/abs/2502.09797v1), 2025-02-13

(66) [TableTalk: Scaffolding Spreadsheet Development with a Language Agent](http://arxiv.org/abs/2502.09787v1), 2025-02-13

(67) [Improving Acoustic Side-Channel Attacks on Keyboards Using Transformers and Large Language Models](http://arxiv.org/abs/2502.09782v1), 2025-02-13

(68) [Prompt and circumstance: A word-by-word LLM prompting approach to interlinear glossing for low-resource languages](http://arxiv.org/abs/2502.09778v1), 2025-02-13

(69) [Knowledge-Enhanced Program Repair for Data Science Code](http://arxiv.org/abs/2502.09771v1), 2025-02-13

(70) [Non-Markovian Discrete Diffusion with Causal Language Models](http://arxiv.org/abs/2502.09767v1), 2025-02-13

(71) [LLM-Generated Microservice Implementations from RESTful API Definitions](http://arxiv.org/abs/2502.09766v1), 2025-02-13

(72) [Enhancing Jailbreak Attacks via Compliance-Refusal-Based Initialization](http://arxiv.org/abs/2502.09755v1), 2025-02-13

(73) [Vote-Tree-Planner: Optimizing Execution Order in LLM-based Task Planning Pipeline via Voting](http://arxiv.org/abs/2502.09749v1), 2025-02-13

(74) [The Widespread Adoption of Large Language Model-Assisted Writing Across Society](http://arxiv.org/abs/2502.09747v1), 2025-02-13

(75) [FoNE: Precise Single-Token Number Embeddings via Fourier Features](http://arxiv.org/abs/2502.09741v1), 2025-02-13

(76) [Making Them a Malicious Database: Exploiting Query Code to Jailbreak Aligned Large Language Models](http://arxiv.org/abs/2502.09723v1), 2025-02-13

(77) [NestQuant: Nested Lattice Quantization for Matrix Products and LLMs](http://arxiv.org/abs/2502.09720v1), 2025-02-13

(78) [Genetic Data Governance in Crisis: Policy Recommendations for Safeguarding Privacy and Preventing Discrimination](http://arxiv.org/abs/2502.09716v1), 2025-02-13

(79) [MME-CoT: Benchmarking Chain-of-Thought in Large Multimodal Models for Reasoning Quality, Robustness, and Efficiency](http://arxiv.org/abs/2502.09621v1), 2025-02-13

(80) [Exploring the Potential of Encoder-free Architectures in 3D LMMs](http://arxiv.org/abs/2502.09620v1), 2025-02-13

(81) [Human-LLM Coevolution: Evidence from Academic Writing](http://arxiv.org/abs/2502.09606v1), 2025-02-13

(82) [SelfCite: Self-Supervised Alignment for Context Attribution in Large Language Models](http://arxiv.org/abs/2502.09604v1), 2025-02-13

(83) [Do LLMs Recognize Your Preferences? Evaluating Personalized Preference Following in LLMs](http://arxiv.org/abs/2502.09597v1), 2025-02-13

(84) [KIMAs: A Configurable Knowledge Integrated Multi-Agent System](http://arxiv.org/abs/2502.09596v1), 2025-02-13

(85) [Logical forms complement probability in understanding language model (and human) performance](http://arxiv.org/abs/2502.09589v1), 2025-02-13

(86) [Polymind: Parallel Visual Diagramming with Large Language Models to Support Prewriting Through Microtasks](http://arxiv.org/abs/2502.09577v1), 2025-02-13

(87) [Zero-shot generation of synthetic neurosurgical data with large language models](http://arxiv.org/abs/2502.09566v1), 2025-02-13

(88) [MDCrow: Automating Molecular Dynamics Workflows with Large Language Models](http://arxiv.org/abs/2502.09565v1), 2025-02-13

(89) [EmbodiedBench: Comprehensive Benchmarking Multi-modal Large Language Models for Vision-Driven Embodied Agents](http://arxiv.org/abs/2502.09560v1), 2025-02-13

(90) [Explainable AI-assisted Optimization for Feynman Integral Reduction](http://arxiv.org/abs/2502.09544v1), 2025-02-13

(91) [Mind the Gap! Choice Independence in Using Multilingual LLMs for Persuasive Co-Writing Tasks in Different Languages](http://arxiv.org/abs/2502.09532v1), 2025-02-13

(92) [Improve LLM-based Automatic Essay Scoring with Linguistic Features](http://arxiv.org/abs/2502.09497v1), 2025-02-13

(93) [Trust at Your Own Peril: A Mixed Methods Exploration of the Ability of Large Language Models to Generate Expert-Like Systems Engineering Artifacts and a Characterization of Failure Modes](http://arxiv.org/abs/2502.09690v1), 2025-02-13

(94) [Objective quantification of mood states using large language models](http://arxiv.org/abs/2502.09487v1), 2025-02-13

(95) [Large Language Models and Provenance Metadata for Determining the Relevance of Images and Videos in News Stories](http://arxiv.org/abs/2502.09689v1), 2025-02-13

(96) [Pixel-Level Reasoning Segmentation via Multi-turn Conversations](http://arxiv.org/abs/2502.09447v1), 2025-02-13

(97) [On multi-token prediction for efficient LLM inference](http://arxiv.org/abs/2502.09419v1), 2025-02-13

(98) [Mind What You Ask For: Emotional and Rational Faces of Persuasion by Large Language Models](http://arxiv.org/abs/2502.09687v1), 2025-02-13

(99) [SQuARE: Sequential Question Answering Reasoning Engine for Enhanced Chain-of-Thought in Large Language Models](http://arxiv.org/abs/2502.09390v1), 2025-02-13

(100) [Truth Knows No Language: Evaluating Truthfulness Beyond English](http://arxiv.org/abs/2502.09387v1), 2025-02-13
